src/daemon/utils.c

```diff
@@ -0,0 +1,15 @@
+#include "utils.h"
+#include "includes.h"
+
+void ckb_generate_random(unsigned int size, unsigned char* data) {
```

Comment:
Also, I think size_t is more appropriate for the size parameter. Then it can be called as such: ckb_generate_random(sizeof(whatever_buffer), whatever_buffer);